### ISSUE TITLE
[!] The 'Pods-Runner' target has transitive dependencies that include statically linked binaries: (/example/ios/Pods/OpenCV-Dynamic-Framework/opencv2.xcframework)

### ISSUE THREAD

**Issue by josephmangmang** [eddy-lau/OpenCV-Dynamic-Framework#2]
[!] The 'Pods-Runner' target has transitive dependencies that include statically linked binaries: (/example/ios/Pods/OpenCV-Dynamic-Framework/opencv2.xcframework)
Do you have idea how to fix this?

**Comment by eddy-lau** [comment]
Try to add this line in your `Podfile`:
```ruby
use_frameworks! :linkage => :static
```

**Comment by josephmangmang** [comment]
> Try to add this line in your `Podfile`:
> 
> ```ruby
> use_frameworks! :linkage => :static
> ```

Yes this one solve the issue and also the `s.static_framework=true` 
but problem with these solutions is I can no longer find my cpp function in Flutter FFI using DynamicLibrary.process()
I'm still looking to solve this static binary issue while maintaining the  cpp function to be accessible in dart.

**Comment by josephmangmang** [comment]
my bad, this package works well when adding to my flutter plugin. No need to make it static framework.

**Comment by josephmangmang** [comment]
![Screenshot 2024-08-14 at 3 29 45 PM](https://github.com/user-attachments/assets/3a8903b7-e6e6-4587-8521-6fe7a4839976)

oh so on  OpenCV-Dynamic-Framework (4.8.0-b1)
issue is still there.

**Comment by josephmangmang** [comment]
OpenCV-Dynamic-Framework (4.5.5)
can be use without error, but 4.8.0-b1 is showing linked binary error
